rules/src/main/scala/fix/UseNamedParameters.scala

```diff
@@ -7,7 +7,7 @@ import scala.annotation.tailrec
 import scala.meta._
 
 final class UseNamedParameters(config: UseNamedParametersConfig)
-    extends SemanticRule(classOf[UseNamedParameters].getSimpleName) {
+    extends SemanticRule(classOf[AbstractSemanticRuleSuite].getSimpleName) {
```

Comment:
looks like we have a false positive and a false negative in https://github.com/scalacenter/scalafix.g8/blob/main/migration-rules/v0.9.28/rules/src/main/scala/fix/v0_9_28.scala cc @mlachkar

Reply:
The rule has been updated to not produce this kind of error. I have tested on this project. I asked if it's possible to re-run scala-steward on this repo. Thanks for reporting it.